openassessment/assessment/api/peer.py

```diff
@@ -45,7 +45,7 @@ def submitter_is_finished(submission_uuid, requirements):
         bool
 
     """
-    if not requirements:
+    if requirements is None:
```

Comment:
Oh yeah good point, I thought we were handling that exception already. I'll update that to match `student_training.py`, but otherwise leave the explicit `None` check.

Thanks!